### PR TITLE
Updated entity code to accomodate varying Google Attributes fields

### PR DESCRIPTION
**Pull request by trob-indie**

### CODE REVIEW

File: event.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ type EventEntity struct {
 	PhotoGallery *[]PhotoGalleryItem `json:"photoGallery,omitempty"`
 	Videos       *[]Video            `json:"videos,omitempty"`
 
-	GoogleAttributes *GoogleAttributes `json:"googleAttributes,omitempty"`
+	GoogleAttributes *map[string]interface{} `json:"googleAttributes,omitempty"`
```
Comment (upalsaha) [inline event.go]: wouldn't it need to be a map of bools instead of interface types?

Comment (trob-indie) [reply event.go]: Json looks like this:

`                "googleAttributes": {
                    "pay_debit_card": [
                        "true"
                    ],
                    "requires_cash_only": [
                        "false"
                    ],
                    "has_gift_wrapping": [
                        "false"
                    ],
                    "has_delivery_same_day": [
                        "false"
                    ]
                },`

Comment (trob-indie) [reply event.go]: So it's actually a map of strings to a list of strings

Comment (upalsaha) [reply event.go]: oof, might want to like change to that then. Should avoid interface types if we know the expected types

Comment (byronharvey24) [reply event.go]: yeah wouldn't it be more accurate to do map[string][]string?

**Review by upalsaha — APPROVED** [review]

File: event.go (hunk 1 above)
Comment (trob-indie) [reply event.go]: yep I updated

**Pull request merged**